test/extended/prometheus/prometheus.go

```diff
@@ -88,6 +88,20 @@ var _ = g.Describe("[sig-instrumentation][Late] Alerts", func() {
 		e2e.Logf("Watchdog alert is firing")
 	})
 
+	g.It("should not have any alerts in pending state the entire cluster run", func() {
+		oc.SetupProject()
+		ns := oc.Namespace()
+		execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
+		defer func() {
+			oc.AdminKubeClient().CoreV1().Pods(ns).Delete(context.Background(), execPod.Name, *metav1.NewDeleteOptions(1))
+		}()
+
+		tests := map[string]bool{
+			`count_over_time(ALERTS{alertstate="pending"}[2h])`: false,
```

Comment:
You need to exclude at least the same alerts as in https://github.com/openshift/origin/blob/master/test/extended/prometheus/prometheus.go#L69